### DISCUSSION TITLE
how to add other wallets / blockchains?

### DISCUSSION THREAD

**Discussion by ryanckulp** [snapshot-labs/snapshot#1725] [Q&A] [answered]
hi folks,

i'd like to fork Snapshot for use on the Klaytn network.

to do this i first need to add a `connector` for the Kaikas wallet, a [Metamask fork](https://docs.kaikas.io/#disclaimer) but for $KLAY token.

is there documentation for how to accomplish this?

Ryan, [@ryanckulp](https://twitter.com/ryanckulp)

**Comment by bonustrack** [comment, marked as answer]
Hi, have a look in the doc here: https://docs.snapshot.org/networks

**Reply by ryanckulp** [reply]
thank you!
